.storybook/preview.ts

```diff
@@ -23,6 +23,8 @@ import docs from "./addons/docs"
 import themeSwitcher from "./addons/theme"
 import withTheme from "./decorators/theme"
 
+import './reset.css'
```

Comment:
why is this reset needed? can you explain in a comment that remains for the future please?

Reply:
I don't know if an extensive comment explaining this is necessary, since it's a fairly common practice in frontend